fiftyone/core/metadata.py

```
@@ -534,6 +590,53 @@ def get_image_info(f):
+        "ffprobe",
+        "-v",
+        "error",
+        "-show_format",
+        "-show_streams",
+        "-of",
+        "json",
+        filepath,
+    ]
+
+    try:
+        res = subprocess.check_output(cmd)
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)